packages/notifi-react-card/lib/utils/NotificationHistoryUtils.tsx

```diff
@@ -135,29 +135,44 @@ supportedEventDetails.set('AccountBalanceChangedEventDetails', {
     const getTitle = () => {
       return detail.direction === 'INCOMING'
         ? `Incoming Transaction: ${changeAmount} ${detail.tokenSymbol}`
-        : `Outgoing Transaction: ${changeAmount} ${detail.tokenSymbol}`;
+        : `Outgoing Transaction: -${changeAmount} ${detail.tokenSymbol}`;
     };
+
+    const getMessage = () => {
+      return detail.direction === 'INCOMING'
+        ? `Wallet ${walletAddress} account balance changed by ${changeAmount} ${detail.tokenSymbol}`
```

Comment:
what about this?

const direction = detail.direction === 'INCOMING' ? '' : '-';

const message = `Wallet ${walletAddress} account balance changed by ${direction}${changeAmount} ${detail.tokenSymbol}`;